migaloo/chain.json

```diff
@@ -0,0 +1,111 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "migaloo",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.whitewhale.money/",
+  "pretty_name": "Migaloo",
+  "chain_id": "migaloo-1",
+  "bech32_prefix": "migaloo",
```

Comment:
Please register this prefix to SLIP173: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Done Pr is here https://github.com/satoshilabs/slips/pull/1497 🤝

Reply:
Following up, this pr is now merged https://github.com/satoshilabs/slips/pull/1497